docs/writing-policies/metadata.md

```diff
@@ -0,0 +1,99 @@
+---
+sidebar_label: "Policy metadata"
+title: "Policy metadata"
+---
+
+# Policy metadata
+
+The Kubewarden metadata file, `metadata.yaml`, serves as a vital configuration
+file that contains important information and settings related to the policies
+used within Kubewarden. This documentation aims to provide an overview of the
+purpose and usage of the metadata file. See an example of a `metadata.yaml`
+file:
+
+```yaml
+rules:
+- apiGroups: [""]
+  apiVersions: ["v1"]
+  resources: ["pods"]
+  operations: ["CREATE"]
+mutating: false
+contextAwareResources: []
+executionMode: kubewarden-wapc
+# Consider the policy for the background audit scans. Default is true. Note the
+# intrinsic limitations of the background audit feature on docs.kubewarden.io;
+# If your policy hits any limitations, set to false for the audit feature to
+# skip this policy and not generate false positives.
+backgroundAudit: true
+annotations:
+  # artifacthub specific:
+  io.artifacthub.displayName: Policy Name
+  io.artifacthub.resources: Pod
+  io.artifacthub.keywords: pod, cool policy, kubewarden
+  io.kubewarden.policy.ociUrl: ghcr.io/myorg/my-policy
+  # kubewarden specific:
+  io.kubewarden.policy.title: My policy
+  io.kubewarden.policy.description: Short description
+  io.kubewarden.policy.author: myself
+  io.kubewarden.policy.url: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.source: https://github.com/yourorg/my-policy
+  io.kubewarden.policy.license: Apache-2.0
+  # The next two annotations are used in the policy report generated by the
+  # Audit scanner. Severity indicates policy check result criticality and
+  # Category indicates policy category. See more here at docs.kubewarden.io
+  io.kubewarden.policy.severity: medium
+  io.kubewarden.policy.category: Resource validation
+```
+
+
+**1. Enabling Background Audit Checks:**
```

Comment:
Just a musing:

Also, as a policy author, I would prefer more of a field by field reference subsections. But maybe that could go into "Reference documentation" in the future. And thinking about it, I feel that it would be useful to separate the docs into the different personas (policy author, policy user, cluster admin). And for each of them, use the Divio system and provide a specific "how-to", "reference documentation", etc.

Reply:
I agree. I think that a field by field documentation is better suited to the reference docs. My goal here is to have some docs about the metadata.ym file. But I do not think this is the defined documentation about that.